template/package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "scripts": {
     "android": "react-native run-android",
-    "ios": "react-native run-ios",
+    "ios": "react-native run-ios --simulator \"iPhone 11 Pro Max\"",
```

Comment:
It seems like run-ios internally should use this simulator as a default if this is desired. Putting it here seems like it is just a hack around the symptom. Also, since it is changing the template it will show up for everyone when they are upgrading.